docs/examples/wpi-many/wpi-many-custom-checker-example.sh

```diff
@@ -24,7 +24,9 @@ checkername=no-literal
 repolist=securerandom.list
 workingdir=$(pwd)
 
-# the stub files for the checker being used
+# The stub files for the checker being used.
+# If you are not using stub files, you MUST comment this line out and modify the last command in
```

Comment:
Rather than modifying the last command, I think this script would be easier to use if there is logic that checks whether the variable is set and provides the `--stubs` option only if the variable is set.  Similarly for `--libs`.

Reply:
I don't disagree that the change you're suggesting would make the script easier to use. However, this script isn't meant to be used - it is an example/template for a user to write their own script. For that reason, the script's logic should be as simple as possible - the "user" is expected to read and understand the entire script, modifying each line for their use-case as they go. No one should run this script without first understanding it and modifying it themselves.

Reply:
The fewer modifications are necessary, the fewer errors users are likely to make.  I think it is better to make the script just a little bit more capable, rather than leaving landmines for less-than-perfect users to stumble over.